exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -194,10 +145,13 @@ func (b *Bitfinex) wsHandleData(respRaw []byte) error {
 		eventType, hasEventType := d[1].(string)
 
 		if chanID != 0 {
-			if c, ok := b.WebsocketSubdChannels[chanID]; ok {
-				return b.handleWSChannelUpdate(c, chanID, eventType, d)
+			if c := b.Websocket.GetSubscription(chanID); c != nil {
+				return b.handleWSChannelUpdate(c, eventType, d)
 			}
-			return fmt.Errorf("unable to locate chanID: %d", chanID)
+			// We didn't have a mapping for this chanID; This probably means we have unsubscribed OR
+			// received our first message before processing the sub chanID
+			// In either case it's okay. No point in erroring because there's nothing we can do about it, and it happens often
+			return nil
```

Comment:
The only critique about this, is that there are potential updates being dropped due to the parallelChanOp which is a way better alternative to what was originally occurring. Just highlighting not a change request.

Reply:
Agreed.
I had this erroring, but it turned out it was a 1000:0 ratio of false positives, so it was just mindless noise.